.github/workflows/scripts/install-and-build-with-sdk.sh

```diff
@@ -73,9 +73,24 @@ if [[ -n "$SWIFT_BUILD_FLAGS" ]]; then
     log "Additional build flags: $SWIFT_BUILD_FLAGS"
 fi
 
+# Detect package manager
+if command -v apt >/dev/null 2>&1; then
+    INSTALL_PACKAGE_COMMAND="apt update -q && apt install -yq"
+elif command -v dnf >/dev/null 2>&1; then
+    INSTALL_PACKAGE_COMMAND="dnf install -y"
+elif command -v yum >/dev/null 2>&1; then
+    INSTALL_PACKAGE_COMMAND="yum install -y"
```

Comment:
nit: the yum branch isn't really needed. yum has been replaced by dnf, and the only distro supported by Swift which hasn't fully moved yet is Amazon Linux 2, but that distro doesn't even work in GitHub actions in general because a number of its constituent components are too old for GitHub to set up its communication pipe

Reply:
Ah that makes sense, I was using `yum` when testing locally on AL2 so I included it, but I didn't realize it wasn't supported for GitHub actions. I think I'll probably keep it in the script for now in case there's a need to use it locally in an AL2 container, e.g. running

```
curl -s --retry 3 https://raw.githubusercontent.com/swiftlang/github-workflows/refs/heads/main/.github/workflows/scripts/install-and-build-with-sdk.sh | \
bash -s -- [--static] [--wasm] [--embedded-wasm] --flags="<build-flags>" <swift-version>
```